action/assert.go

```diff
@@ -75,7 +75,7 @@ func (assert *AssertOpts) verify(eio base.Driver) (bool, error) {
 
 	// check assert conditions
 	for {
-		exp, err := base.BuildExperiment(true, eio)
+		exp, err := base.BuildExperiment(eio)
```

Comment:
Experiment build will always read results. But they may or may not be reused during the run (based on the --reuseResult flag). For looped cron-job experiments, `reuseResult` is true.